components/brave_new_tab_ui/components/default/widget/widgetMenu.tsx

```diff
@@ -15,6 +15,12 @@ import LearnMoreIcon from './assets/learn-more'
 import FavoritesIcon from './assets/favorites'
 import { getLocale } from '../../../../common/locale'
 
+export interface WidgetMenuCustomItem {
+  label: string
+  renderIcon: () => React.ReactElement<{}>
```

Comment:
This type could probably be the more general `React.ReactNode`.

Reply:
Thank you, I suspected what I had there was not quite right! Fixed.